src/Libraries/DSIronPython/IronPythonCodeCompletionProviderCore.cs

```diff
@@ -219,7 +217,6 @@ private Type TryGetTypeFromFullName(string name)
             for (var i = 0; i < matches.Count; i++)
             {
                 var wholeLine = matches[i].Groups[0].Value.TrimEnd('\r', '\n');
-                var libName = matches[i].Groups[3].Value.Trim();
```

Comment:
Is it ok that this is not needed? Who would know?

Reply:
the local var seems to be copy paste error from another similar method.